packages/lodestar/src/network/gossip/validatorFns/index.ts

```diff
@@ -0,0 +1,16 @@
+import {GossipType} from "../interface";
+import {validateAggregatedAttestation} from "./aggregatedAttestation";
+import {validateCommitteeAttestation} from "./attestation";
+import {validateAttesterSlashing} from "./attesterSlashing";
+import {validateBeaconBlock} from "./block";
+import {validateProposerSlashing} from "./proposerSlashing";
+import {validateVoluntaryExit} from "./voluntaryExit";
```

Comment:
we could rexport them since I believe we are reusing them in api?

Reply:
chain `packages/lodestar/src/chain/validation/*` exports the actual validator functions. Then the API and this `network/gossip/validatorFns/*` consume those. Actually the functions being moved here only do error handling of the validation but not the validation itself.